samples/CommunityToolkit.Maui.Markup.Sample/ViewModels/SettingsViewModel.cs

```diff
@@ -15,16 +17,8 @@ public SettingsViewModel(SettingsService settingsService)
 		NumberOfTopStoriesToFetch = settingsService.NumberOfTopStoriesToFetch;
 	}
 
-	public int NumberOfTopStoriesToFetch
```

Comment:
I must be missing this detail. Does the MVVM toolkit provide the magic of calling this for us?

Reply:
Yup! The generated `partial class` provides `partial` methods like these that we can (and are) leverage:

- `partial void OnNumberOfTopStoriesToFetchChanged()`
- `partial void OnNumberOfTopStoriesToFetchChanging()`